src/js/components/Audio/Audio.js

```diff
@@ -0,0 +1,276 @@
+import React, { Component } from 'react';
+import { compose } from 'recompose';
+
+import { withTheme } from 'styled-components';
+
+import { Box } from '../Box';
+import { Button } from '../Button';
+import { RangeInput } from '../RangeInput';
+import { Text } from '../Text';
+import { withForwardRef } from '../hocs';
+
+import { throttle, formatTime, mediaEvents } from '../../utils';
+
+import {
+  StyledAudio,
+  StyledAudioContainer,
+  StyledAudioControls,
+} from './StyledAudio';
+
+// default value for the volume can range from 0 to 1. (HTML audio defaults to 1 i.e. max volume)
+const INITIAL_VOLUME = 0.3;
```

Comment:
Does this mean that 1.0 matches the volume that the user has set for his speakers? I would think we should align to that.

Reply:
Yes, and when you put it like that it makes more sense to keep `1.0` as the default. I'll remove the associated logic that make it more custom.